components/builder-web/app/shared/_shared.module.scss

```diff
@@ -3,8 +3,11 @@
 @import 'channels/channels.component';
 @import 'checking-input/checking-input.component';
 @import "copyable/copyable.component";
-@import "docker-export-settings/docker-export-settings.component";
+@import 'docker-export-settings/docker-export-settings.component';
```

Comment:
Is it supposed to be single or double? The one above is also double

Reply:
Ah, missed one, thanks. (I like single, generally.)